css-canvas-medical/styles.css

```diff
@@ -0,0 +1,229 @@
+.container {
+
+}
+
+p {
+  font-family: 'Avenir';
+}
+
+.navBar {
+  display: flex;
+  border-bottom: 1px solid #c7c6c3;
+  margin-top: 40px;
+}
+
+.blueLogoContainer {
+  margin-left: 80px;
+  text-align: left;
+}
+
+.logo {
+  width: 50%;
+  margin-bottom: 20px;
+}
+
+.navBarItems {
+  margin: auto;
+  display: flex;
+  margin-right: 80px;
+  font-size: 12px;
+}
+
+.navBarlink {
+  margin: auto 20px;
+}
+
+a {
+  text-decoration: none;
+  color: #999896;
+}
+
+#contactButton {
+  font-size: 12px;
+  padding: 10px;
+  text-align: center;
+  background-color: #036;
+  color: white;
+  border: none;
+  border-radius: 3px;
+  width: 90px;
+}
+
+.emrPic {
+  width: 1100px;
+  height: 500px;
+  background-image: url('images/hero.jpg');
+  background-size: cover;
+  background-position: center;
+  position: relative;
+  margin: auto;
+  margin-top: 55px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+  width: 1100px;
+  height: 500px;
+}
+
+.overlayTextContent {
+  margin: 50px;
+  position: absolute;
+  bottom: 100px;
+  line-height: 30px;
+  color: white;
+  width: 50%;
+}
+
+#title {
+  font-family: 'Tiempos Headline';
+  font-weight: 400;
+  width: 80%;
+}
+
+#paragraph {
+  font-family: 'Avenir';
+  font-size: 13px;
+}
+
+.startButton {
+  font-family: 'Avenir';
+  font-size: 14px;
+  padding: 12px;
+  text-align: center;
+  background-color: #036;
+  color: white;
+  border: none;
+  border-radius: 3px;
+  width: 150px;
+  position: absolute;
+  margin-top: 10px;
+}
+
+.cliniciansContainer {
+  margin: auto;
+  margin-top: 60px;
+  background-color: #f1f1f1;
+  text-align: center;
+  width: 90%;
+  height: 400px;
+  color: #4b4b4b;
+}
+
+.clinicians {
+  position: relative;
+  top: 40px;
```

Comment:
Rather than reach for positioning, try leveraging the box model of the parent element instead (padding).